### ISSUE TITLE
disallow submisison of repeated files.

### ISSUE THREAD

**Issue by heitorPB** [exercism/cli#900]
Currently, the `exercism` cli allows us to submit the same file twice as a solution:

```
exercismo submit afile.cpp afile.cpp
```

I just did that accidentally instead of the `.cpp` and `.h`. The solution page lists twice the same file.
is this a feature or a bug?

**Comment by sshine** [comment]
If it's a feature, it is probably not useful.

If it's a bug it has a low impact.

We should probably give it a low priority and fix it.

**Comment by kytrinyx** [comment]
Yeah, this is a bug. I agree with @sshine that we should fix it, but it's not a dire one, so not something we need to scramble to fix.

Thanks @heitorPB, for reporting it!

**Comment by carlows** [comment]
I was having a look at this but couldn't reproduce it, I tried submitting the same file twice over an elixir track and the solution page listed it only once. 

I also saw this function in the code that is removing duplicate paths before the submit https://github.com/exercism/cli/blob/master/cmd/submit.go#L150-L162

@heitorPB does this still happen to you?

**Comment by heitorPB** [comment]
I'm using exercism 3.0.10, in the cpp track:

```
$ exercism submit say.cpp say.cpp


    Your solution has been submitted successfully.
    View it at:

    
    https://exercism.io/my/solutions/bfee4819815244fdb102ebba99fcc592
```

And the website shows:

![say](https://user-images.githubusercontent.com/13461702/70457216-82c75080-1a8e-11ea-8138-785cca65ed04.png)

I just tested it and still happens.

**Comment by carlows** [comment]
@heitorPB I think that what you described was fixed in a later version, the current version of the exercism CLI is 3.0.13. 

Can you please try updating to the latest version? 😄

**Comment by heitorPB** [comment]
With CLI 3.0.13, only one file appears in the website!
